backend/app/api/question.py

```
@@ -0,0 +1,195 @@
+    Args:
+        question_in (QuestionCreate): The data for creating a new question.
+        session (AsyncSession): Database session provided by FastAPI's dependency injection system.
+        current_user (User): The authenticated user.
+
+    Returns:
+        Question: The newly created question.
+    """
+    try:
+        quiz = await get_quiz(session=session, quiz_id=question_in.quiz_id)
+        if quiz.owner_id != current_user.id:
+            raise QuizPermissionException("Not authorized to add a question to this quiz")
```

Ruff (0.14.1)

Abstract `raise` to an inner function

(TRY301)